src/crimson/tools/perf_staged_fltree.cc

```diff
@@ -13,6 +13,7 @@
 #include "crimson/os/seastore/onode_manager/staged-fltree/node_extent_manager.h"
 
 #include "test/crimson/seastore/onode_tree/test_value.h"
+#include "test/crimson/gtest_seastar.h"
```

Comment:
So we were missing that `perf_staged_fltree` also uses `EphemeralTestState` which was impacted by https://github.com/ceph/ceph/pull/53232.

So `EphemeralTestState` should tolerate that `INSTANTIATE_TEST_SUITE_P` is not defined in the non-test scenario like this, probably by defauting the j_type to "segmented" ?